robot/impl/resource_manager.go

```diff
@@ -830,20 +848,40 @@ func (manager *resourceManager) updateResources(
 	defer manager.configLock.Unlock()
 	var allErrs error
 
+	for _, p := range conf.Added.Packages {
+		if err := manager.validateUniqueResource(p.Package); err != nil {
```

Comment:
Ah what if a package and a module have the same name? I don't think we even need to validate uniqueness of packages here, we can probably let package manager do it themselves.

Reply:
hm, i don't think that would be an issue because package.package must be unique as it includes orgID:type:name. If I was using name I would agree, but i don't really think its possible unless its an explicit error.

Reply:
Ah gotcha; this seems good then! Thanks for explanation + ignore my other comment about leaving package uniqueness to package managers. Might need a test for non-unique package names, too, then.